### PR TITLE
Add quantum computer policy paper

### PR DESCRIPTION
**Pull request by nathanshammah**
Add quantum computer policy paper.

### CODE REVIEW

File: src/pages/research/publications.mdx [hunk 1]
```diff
@@ -34,6 +34,7 @@ The Unitary Fund [research GitHub repo](https://github.com/unitaryfund/research)
 
 ## Other Unitary Fund staff research
 
+- Travis L. Scholten, Carl J. Williams, Dustin Moody, Michele Mosca, William Hurley ("whurley"), William J. Zeng, Matthias Troyer, Jay M. Gambetta, *Assessing the Benefits and Risks of Quantum Computers* [[2401.16317](https://arxiv.org/abs/2401.16317)]
```
Comment (nathanshammah) [inline src/pages/research/publications.mdx:37]:
```suggestion
- Travis L. Scholten, Carl J. Williams, Dustin Moody, Michele Mosca, William Hurley ("whurley"), William J. Zeng, Matthias Troyer, Jay M. Gambetta, *Assessing the Benefits and Risks of Quantum Computers*, arXiv (2024), 
-  [[2401.16317](https://arxiv.org/abs/2401.16317)]
```

File: src/pages/research/publications.mdx [hunk 2]
```diff
@@ -34,6 +34,8 @@ The Unitary Fund [research GitHub repo](https://github.com/unitaryfund/research)
 
 ## Other Unitary Fund staff research
 
+- Travis L. Scholten, Carl J. Williams, Dustin Moody, Michele Mosca, William Hurley ("whurley"), William J. Zeng, Matthias Troyer, Jay M. Gambetta, *Assessing the Benefits and Risks of Quantum Computers*, arXiv (2024), 
+-  [[2401.16317](https://arxiv.org/abs/2401.16317)]
```
Comment (Misty-W) [inline src/pages/research/publications.mdx:38]:
```suggestion
- Travis L. Scholten, Carl J. Williams, Dustin Moody, Michele Mosca, William Hurley ("whurley"), William J. Zeng, Matthias Troyer, Jay M. Gambetta, *Assessing the Benefits and Risks of Quantum Computers*, arXiv (2024), [[2401.16317](https://arxiv.org/abs/2401.16317)]
```

Comment (Misty-W) [reply src/pages/research/publications.mdx:38]: I believe this is the intended formatting.

**Review by Misty-W — APPROVED** [review]
Great to have this added, look forward to diving in deeper soon! 

I suggested a formatting fix, otherwise looks good.

**Comment by nathanshammah** [comment]
Thanks @Misty-W, indeed!

**Pull request merged**